src/network/readme.md

```diff
@@ -0,0 +1,116 @@
+# Overview
+
+LRS-Net 3.0 is the LRS subsystem which enables access to RealSense cameras over the network. It including following features:
+
+* Server-side software upgrade support.
+* Firmware update for the remote camera.
+* Set of LRS-Net 2.0 features, with the following majors to highlight:
```

Comment:
User are not familiar with LRS-Net 2.0,
Ether make it a link or replace it with all features list.